qa/suites/upgrade/nautilus-x/parallel/3-upgrade-sequence/upgrade-mon-osd-mds.yaml

```diff
@@ -45,3 +46,8 @@ upgrade-sequence:
        wait-for-healthy: true
    - sleep:
        duration: 60
+   - ceph.restart:
+       daemons: [client.1]
```

Comment:
upgrade-all.yaml should also restart client.1, probably along with `daemons: [mds.a]`

Reply:
you are right, added